examples/android-activity-graphs/build.gradle

```diff
@@ -0,0 +1,26 @@
+apply plugin: 'com.android.application'
+
+dependencies {
+}
+
+android {
+    compileSdkVersion defaultCompileSdkVersion
+    buildToolsVersion defaultBuildToolsVersion
+
+    defaultConfig {
+        minSdkVersion defaultMinSdkVersion
+        targetSdkVersion defaultTargetSdkVersion
+    }
+
+    lintOptions { abortOnError false }
+}
+
+dependencies {
+    compile "com.android.support:appcompat-v7:${defaultSupportLibVersion}"
+    compile "com.android.support:design:${defaultSupportLibVersion}"
+
+    compile "com.google.dagger:dagger:$dagger_version"
+    annotationProcessor "com.google.dagger:dagger-compiler:$dagger_version"
+    provided 'org.glassfish:javax.annotation:10.0-b28'
```

Comment:
Should there be a comment explaining why Android projects need that dependency? (furthermore as it's not needed for Maven-built Android projects)